beta-src/src/utils/state/gameApiSlice/extraReducers/fetchGameData/fulfilled.ts

```diff
@@ -46,47 +47,10 @@ export default function fetchGameDataFulfilled(state, action): void {
       state.ownUnits.push(unit.id);
     }
   });
-  const unitsToDraw = getUnits(data, members);
-  Object.values(data.territories).forEach(({ name }) => {
-    const mappedTerritory = TerritoryMap[name];
-    const command: GameCommand = {
-      command: "SET_UNIT",
-      data: { setUnit: { unitSlotName: mappedTerritory.unitSlotName } },
-    };
-    setCommand(
-      state,
-      command,
-      "territoryCommands",
-      mappedTerritory.parent
-        ? Territory[mappedTerritory.parent]
-        : Territory[mappedTerritory.territory],
-    );
-  });
-  unitsToDraw.forEach(({ country, mappedTerritory, unit }) => {
-    const command: GameCommand = {
-      command: "SET_UNIT",
-      data: {
-        setUnit: {
-          componentType: "Game",
-          country,
-          mappedTerritory,
-          unit,
-          unitType: unit.type as UnitType,
-          unitSlotName: mappedTerritory.unitSlotName,
-        },
-      },
-    };
-    setCommand(
-      state,
-      command,
-      "territoryCommands",
-      mappedTerritory.parent
-        ? Territory[mappedTerritory.parent]
-        : Territory[mappedTerritory.territory],
-    );
-  });
+  state.units = getUnits(data, members);
 
   state.territoriesMeta = getTerritoriesMeta(data);
-  highlightMapTerritoriesBasedOnStatuses(state);
+
+  // FIXME: figure out what's here
   updateOrdersMeta(state, getOrdersMeta(data, board, phase));
```

Comment:
What does the FIXME here mean? 

updateOrdersMeta is used to update the `ordersMeta` object, which is like a draft of the orders (which differ from the ones that came in from the server on initial load). This is what changes as the user submits new moves on the map. When the user saves it takes these draft orders and uses the data to mutate the orders that are then sent to the server.

Reply:
`ordersMeta` is referenced to know how to draw the different orders (convoy, support, move, etc). 

When the user clicks "save" it does the aforementioned inside the clickButton function in WDMoveControls.tsx

Reply:
I think the FIXME was for myself around "when should we be clobbering the current orders with incoming data from the server?" I.e. you can observe sometimes you enter orders and then if you don't click "Save" quickly enough they will get overrident from the server.

I fixed that with a follow-up commit in this PR, by only calling `updateOrdersMeta` if there are no unsaved orders.